Phishing Campaign - GetCampaignIncidentsInfo handling deleted incidents (#27621)
demisto/content: Packs/Campaign/ReleaseNotes/3_2_24.md

#### Scripts

##### GetCampaignIncidentsInfo

> - Fixed an issue where ***GetCampaignIncidentsInfo*** was failed when some incidents were deleted.

Reviewer: ```suggestion
- Fixed an issue causing the script to fail when incidents were deleted.
```
